modules/push-notifications/src/main/native/android/dalvik/PushFcmMessagingService.java

```diff
@@ -136,26 +131,46 @@ public void onComplete(Task<String> task) {
         });
     }
 
-    private void sendNotification(String id, String title, String body) {
+    private void sendNotification(HashMap<String, String> payload) {
         NotificationManager notificationManager =
                 (NotificationManager) getSystemService(Activity.NOTIFICATION_SERVICE);
         if (debug) {
-            Log.v(TAG, "Sending push notification with id: " + id + ", title: " + title + ", body: " + body);
+            Log.v(TAG, "Sending push notification with payload: " + jsonPrintMap(payload));
+        }
+	int requestCode = (int) (System.currentTimeMillis() % Integer.MAX_VALUE);
+        notificationManager.notify(requestCode, getNotification(requestCode, payload));
+    }
+
+    private String jsonPrintMap(HashMap<String, String> map, String... keys) {
```

Comment:
The `keys` parameter is no longer being used. Removing it will simplify the method.